website/manual.md

```diff
@@ -2,89 +2,88 @@
-  it), ought to also be able to be run in a modern web browser without change.
+Deno aims to be a productive and secure scripting environment for the modern programmer.
 
-- [Aims to support top-level `await`.](https://github.com/denoland/deno/issues/471)
+Deno will always be distributed as a single executable. Given a URL to a Deno program, it is runnable with nothing more than [the 10 megabyte zipped executable](https://github.com/denoland/deno/releases). Deno explicitly takes on the role of both runtime and package manager. It uses a standard browser-compatible protocol for loading modules: URLs.
 
-- Be able to serve HTTP efficiently.
-  ([Currently it is relatively slow.](https://deno.land/benchmarks.html#req-per-sec))
+Among other things, Deno is a great replacement for utility scripts that may have been historically written with bash or python.
 
-<!-- prettier-ignore-start -->
-<!-- see https://github.com/prettier/prettier/issues/3679 -->
+### Goals
 
-- Provide useful tooling out of the box:
-    - dependency inspector (`deno info`)
-    - code formatter (`deno fmt`),
-    - bundling (`deno bundle`)
-    - runtime type info (`deno types`)
-    - test runner (`deno test`)
-    - command-line debugger (`--debug`)
-      [not yet](https://github.com/denoland/deno/issues/1120)
-    - linter (`deno lint`) [not yet](https://github.com/denoland/deno/issues/1880)
+1. Only ship a single executable (`deno`).
+2. Provide Secure Defaults
+   * Unless specifically allowed, scripts can't access files, the environment, or the network.
+3. Browser compatible: The subset of Deno programs which are written completely in JavaScript and do not use the global `Deno` namespace (or feature test for it), ought to also be able to be run in a modern web browser without change.
+4. Be able to serve HTTP efficiently
+5. Provide a great developer experience include built-in tooling.
+6. Does not leak browser or V8 concepts into user land.
+
+### Comparison to Node.js
+
+* Deno does not use `npm`
+  * It uses modules referenced as URLs or file paths
+* Deno does not use a `package.json`
+  * This is a non-goal. There are effective patterns [citation needed] for managing dependencies.
+* Deno provides different APIs than node.
+* Deno requires explicit permissions for file, network, and environment access.
+  * Node is less secure out of the box.
+* Deno always dies on uncaught errors.
+* Uses "ES Modules" and does not support `require()`. Like the browser, allows imports from URLs:
 
-<!-- prettier-ignore-end -->
+```ts
+import * as log from "https://deno.land/std/log/mod.ts";
+```
 
-### Non-goals
+### Other key behaviors
 
-- No `package.json`.
+* Remote code is fetched and cached on first execution, and never updated until the code is run with the `--reload` flag. (So, this will still work on an airplane. See `~/.deno/src` for details on the cache.)
+* Modules/files loaded from remote URLs are intended to be immutable and cacheable.
```

Comment:
I'm not sure if this is the right home for these points, but I didn't want to delete them.

I also added the note about immutable intention which @kitsonk and I talked about a bit. It's an opinionated proposal, but worth discussing.

Reply:
Also mentioned this in https://github.com/denoland/deno/issues/3158